### DISCUSSION TITLE
Seed database using xls file

### DISCUSSION THREAD

**Discussion by younes-alouani** [prisma/prisma#12865] [Q&A]
How can I seed my database using an excel file (xls) ?

**Comment by younes-alouani** [comment]
Dear community,

I want to convert an excel file into a list of javascript dictionaries.

Here is a sample of the excel file (more than 10 000 rows) :

id	category	                description
1	Open Source	        Fullstack React framework
2	Closed Source	Next Generation ORM

Here is the list of dictionaries I would like to have : in order to use seed.ts

```
[
    {
        id : 1,
        category : "Open Source",
        description : "Fullstack React framework",
    },
    {
        id : 2,
        category : "Closed Source",
        description : "Next Generation ORM ",
    }

]
```

**Comment by austincrim** [comment]
Hey @younes-alouani 👋 ,

You will need to manually read and parse the .xls file in your `seed.ts` script. You could potentially use an NPM package like [this one](https://www.npmjs.com/package/read-excel-file).

**Reply by younes-alouani** [reply]
Hey @austincrim 
Thank you for your answer.
At the end, i converted the xls to sql file and used psql command to dump the database.

**Comment by ludralph** [comment]
Hi there,

To keep our discussions organized and focused on the most relevant topics, we’re reviewing and tidying up our backlog. As part of this process, we’re closing discussions that haven’t had any recent activity and appear to be outdated.

If this discussion is still important to you or unresolved, we’d love to hear from you! Feel free to reopen it or start a new one with updated details.

For more details about our priorities and vision for the future of Prisma ORM, check out our latest blog post: https://www.prisma.io/blog/prisma-orm-manifesto.

Thank you for your understanding and being part of the community!
